ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ConfigManagementServiceIT.scala

```diff
@@ -158,7 +159,21 @@ final class ConfigManagementServiceIT extends LedgerTestSuite {
         response1.configurationGeneration + 1 == response2.configurationGeneration,
         s"New configuration's generation (${response2.configurationGeneration} should be original configurations's generation (${response1.configurationGeneration} + 1) )",
       )
-      assertGrpcError(failure, Status.Code.ABORTED, "")
+      failure match {
```

Comment:
If you want, you could probably try to be clever and do something like
```
Try(asserGrpcError(failure, Status.Code.ABORTED, "")).getOrElse(assertGrpcError(failure, Status.Code.INVALID_ARGUMENT, ""))
```
Looks cute but I think your version is probably easier to read.

Reply:
oh so simple! :man_facepalming: and even no additional dependency!
but I would still stay with the original if it is fine with you because:
* readability
* more precise error reporting